spec/features/admin/invoices/index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Invoices Index Spec' do 
+  before :each do 
+    @merchant1 = Merchant.create!(name: "Kevin's Illegal goods")
+    @merchant2 = Merchant.create!(name: "Denver PC parts")
+
+    @customer1 = Customer.create!(first_name: "Sean", last_name: "Culliton")
+    @customer2 = Customer.create!(first_name: "Sergio", last_name: "Azcona")
+    @customer3 = Customer.create!(first_name: "Emily", last_name: "Port")
+
+    @item1 = @merchant1.items.create!(name: "Funny Brick of Powder", description: "White Powder with Gasoline Smell", unit_price: 5000)
+    @item2 = @merchant1.items.create!(name: "T-Rex", description: "Skull of a Dinosaur", unit_price: 100000)
+    @item3 = @merchant2.items.create!(name: "UFO Board", description: "Out of this world MotherBoard", unit_price: 400)
+
+    @invoice1 = Invoice.create!(status: 1, customer_id: @customer2.id, created_at: "2022-11-01 11:00:00 UTC")
+    @invoice2 = Invoice.create!(status: 1, customer_id: @customer1.id, created_at: "2022-11-01 11:00:00 UTC")
+    @invoice3 = Invoice.create!(status: 1, customer_id: @customer3.id, created_at: "2022-11-01 11:00:00 UTC")
+    
+    @invoice_item1 = InvoiceItem.create!(quantity: 1, unit_price: 5000, status: 0, item_id: @item1.id, invoice_id: @invoice1.id)
+    @invoice_item2 =InvoiceItem.create!(quantity: 2, unit_price: 5000, status: 1, item_id: @item2.id, invoice_id: @invoice1.id)
+    @invoice_item3 = InvoiceItem.create!(quantity: 54, unit_price: 8000, status: 2, item_id: @item3.id, invoice_id: @invoice2.id)
+
+  end
+
+  describe 'Creating Admin Invoice Index' do 
+    it 'when I visit admin invoice index, I see a list of all invoice ids in the system' do 
+      visit admin_invoices_path
+      expect(page).to have_content(@invoice1.id)
+      expect(page).to have_content(@invoice2.id)
+      expect(page).to have_content(@invoice3.id)
+    end
+    it "each ID is a link to the admin invoice show page" do 
+      visit admin_invoices_path
+      expect(page).to have_link(@invoice1.id)
+      expect(page).to have_link(@invoice2.id)
+      expect(page).to have_link(@invoice3.id)
+
+      click_link(@invoice1.id)
+      expect(current_path).to eq(admin_invoice_path(@invoice1))
+    end
+  end
```

Comment:
consider including a _`to_not`_ in tests just to confirm that an alternative is not accepted